### PR TITLE
Added basic start button and started watch

### PR DESCRIPTION
**Pull request by kschultee**

### CODE REVIEW

**Review by ronperris — CHANGES_REQUESTED** [review]
Can you style the page?

File: startTimer/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Timer Start</title>
+    <link rel="stylesheet" href="style.css"/>
+  </head>
+  <body>
+    <p><span id = "mins">00</span><span>:</span><span id = "seconds">00</span><span>:</span><span id = "tenths">00</span></p>
+    <div style="text-align:center;">
```
Comment (ronperris) [inline startTimer/index.html]: Don't inline style. Move this to the style sheet.

File: startTimer/index.html [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    <button id="start">Start</button>
```
Comment (ronperris) [inline startTimer/index.html]: This should be indented.

File: startTimer/main.js [hunk 3]
```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-unused-vars */
+var tenths = 0
+var seconds = 0
+var mins = 0
+var getTenths = document.getElementById('tenths')
+var getSeconds = document.getElementById('seconds')
+var getMins = document.getElementById('mins')
+var start = document.getElementById('start')
+var interval
```
Comment (ronperris) [inline startTimer/main.js]: There should be a new line here.

File: startTimer/main.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+start.onclick = function () {
+  interval = setInterval(startTimer, 10)
+}
```
Comment (ronperris) [inline startTimer/main.js]: There should be a new line here.

File: startTimer/main.js [hunk 5]
```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-unused-vars */
+var tenths = 0
```
Comment (ronperris) [inline startTimer/main.js]: Move these related values into an object.

File: startTimer/main.js [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+var mins = 0
+var getTenths = document.getElementById('tenths')
+var getSeconds = document.getElementById('seconds')
+var getMins = document.getElementById('mins')
+var start = document.getElementById('start')
+var interval
+start.onclick = function () {
+  interval = setInterval(startTimer, 10)
+}
+function startTimer() {
```
Comment (ronperris) [inline startTimer/main.js]: Pass the timer object you created above into this function.

**Review by ronperris — CHANGES_REQUESTED** [review]

**Review by ghost — APPROVED** [review]

**Pull request merged**